src/utils/Router.js

```diff
@@ -30,7 +30,7 @@ export default function Routes() {
             <Route path="/" element={<Navigate to="/certificate?create=y" />} />
             <Route path="/certificate" element={<Outlet />}>
               <Route index element={<CreateViewSwitcher />} />
```

Comment:
do we also need createviewswitcher since we are removing manager?

Reply:
Well, this line `<Route index element={<CreateViewSwitcher />} />` represent a way to sort out two routes , 

the first being **`/certificate?create=y`** while the second is just **`/certificate`** which renders all the certificates

I was not able to do it on two lines unfortunately ( this is because we are using `react-router-dom` v6 .. there are some limitations when it comes to this version and all search params like ?create=xyz need to be dealt separately ).

I even tried this but didn't work didn't work :

```js
              <Route index element={(function F() {
                  const create = useSearchParams()[0].get('create')
                  return create ? <Certificates /> : <CertificatesViewAll />
                })()}
              />
```

So I was left with one option which is what you can see in the code